docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -106,4 +106,17 @@ However, this property is deprecated and will be removed in future releases, so
 The management interface uses a different HTTP server than the default {project_name} HTTP server, and it is possible to configure them separately.
 Beware, if no values are supplied for the management interface properties, they are inherited from the default {project_name} HTTP server.
 
-For more details, see https://www.keycloak.org/server/management-interface[Configuring the Management Interface].
\ No newline at end of file
+For more details, see https://www.keycloak.org/server/management-interface[Configuring the Management Interface].
+
+= Escaping slashes in group paths
+
+{project_name} has never escaped slashes in the group paths. Because of that a group named `group/slash` child of `top` uses the full path `/top/group/slash`, which is clearly misleading. Since this version the server can be started to perform escaping of those slashes in the name:
```

Comment:
```suggestion
{project_name} has never escaped slashes in the group paths. Because of that, a group named `group/slash` child of `top` uses the full path `/top/group/slash`, which is clearly misleading. Since this version the server can be started to perform escaping of those slashes in the name:
```